src/scss/config/color/_index.scss

```diff
@@ -25,6 +25,21 @@
   tools.compile-colors(meta.module-variables('ccs-defaults'))
 );
 
+/// ## Content Colors
+/// --------------
+/// Rather than using the brand colors directly,
+/// we use them to set default values for generated variables.
+/// This provides a mapping between semantic foreground use-cases,
+/// and automatically-generated color variables.
+/// @group config-color
+/// @example scss
+///   @use 'config/color/content';
+///   @use 'config/tools';
+///   @use 'sass:meta';
+///
+///   @each $name, $color in tools.compile-colors(meta.module-variables('content')) {
+///   /*! #{$name}: #{$color}; */
+///   }
 @include herman.add(
   'colors',
   'content-colors',
```

Comment:
@mirisuzanne doesn't this `content-colors` here reference the `/// @colors content-colors` line that we no longer have from the `src/scss/config/color/_content.scss` file?

Reply:
uh, yes, I guess so - that line is adding the colors to Herman, even though (for now) we're not displaying them with Herman. So it's not doing anything useful, but also not causing any issues. We could remove that `herman.add` for now, or just leave it so it works once we add custom property support to Herman? Either way.

Reply:
OK, I'd say we can leave it there for now since it doesn't hurt anything and maybe if/when we update Herman we will be in good shape.